src/appkit.cs

```diff
@@ -21624,6 +21624,919 @@ partial interface NSAccessibilityAttributes {
+
+		[Field ("NSAccessibilityWindowMiniaturizedNotification")]
+		NSString WindowMiniaturizedNotification { get; }
+
+		[Field ("NSAccessibilityWindowDeminiaturizedNotification")]
+		NSString WindowDeminiaturizedNotification { get; }
+
+		[Field ("NSAccessibilityDrawerCreatedNotification")]
+		NSString DrawerCreatedNotification { get; }
+
+		[Field ("NSAccessibilitySheetCreatedNotification")]
+		NSString SheetCreatedNotification { get; }
+
+		[Field ("NSAccessibilityUIElementDestroyedNotification")]
+		NSString UIElementDestroyedNotification { get; }
+
+		[Field ("NSAccessibilityValueChangedNotification")]
+		NSString ValueChangedNotification { get; }
+
+		[Field ("NSAccessibilityTitleChangedNotification")]
+		NSString TitleChangedNotification { get; }
+
+		[Field ("NSAccessibilityResizedNotification")]
+		NSString ResizedNotification { get; }
+
+		[Field ("NSAccessibilityMovedNotification")]
+		NSString MovedNotification { get; }
+
+		[Field ("NSAccessibilityCreatedNotification")]
+		NSString CreatedNotification { get; }
+
+		[Mac (10, 9)]
+		[Field ("NSAccessibilityLayoutChangedNotification")]
+		NSString LayoutChangedNotification { get; }
+
+		[Field ("NSAccessibilityHelpTagCreatedNotification")]
+		NSString HelpTagCreatedNotification { get; }
+
+		[Field ("NSAccessibilitySelectedTextChangedNotification")]
+		NSString SelectedTextChangedNotification { get; }
+
+		[Field ("NSAccessibilityRowCountChangedNotification")]
+		NSString RowCountChangedNotification { get; }
+
+		[Field ("NSAccessibilitySelectedChildrenChangedNotification")]
+		NSString SelectedChildrenChangedNotification { get; }
+
+		[Field ("NSAccessibilitySelectedRowsChangedNotification")]
+		NSString SelectedRowsChangedNotification { get; }
+
+		[Field ("NSAccessibilitySelectedColumnsChangedNotification")]
+		NSString SelectedColumnsChangedNotification { get; }
+
+		[Mac (10, 6)]
+		[Field ("NSAccessibilityRowExpandedNotification")]
+		NSString RowExpandedNotification { get; }
+
+		[Mac (10, 6)]
+		[Field ("NSAccessibilityRowCollapsedNotification")]
+		NSString RowCollapsedNotification { get; }
+
+		[Mac (10, 6)]
+		[Field ("NSAccessibilitySelectedCellsChangedNotification")]
+		NSString SelectedCellsChangedNotification { get; }
+
+		[Mac (10, 6)]
+		[Field ("NSAccessibilityUnitsChangedNotification")]
+		NSString UnitsChangedNotification { get; }
+
+		[Mac (10, 6)]
+		[Field ("NSAccessibilitySelectedChildrenMovedNotification")]
+		NSString SelectedChildrenMovedNotification { get; }
+
+		[Mac (10, 7)]
+		[Field ("NSAccessibilityAnnouncementRequestedNotification")]
+		NSString AnnouncementRequestedNotification { get; }
+	}
+
+	[Static]
+	public interface NSWorkspaceAccessibilityNotifications {
+		[Mac (10, 10)]
+		[Field ("NSWorkspaceAccessibilityDisplayOptionsDidChangeNotification")]
+		NSString DisplayOptionsDidChangeNotification { get; }
+	}
+
+	[Static]
+	public interface NSAccessibilityNotificationUserInfoKeys {
+		[Field ("NSAccessibilityUIElementsKey")]
```

Comment:
But I think other custom keys can be used in the UserInfo dictionary too, so making a strong dictionary wouldn't be possible.